docs/guides/deploy_service.md

```diff
@@ -1,24 +1,34 @@
 Deploying a service with the {{open_autonomy}} framework requires that you have an already available service configuration, which you could be a service created from scratch, fetched from the [IPFS](https://ipfs.io/), or already registered in the on-chain protocol.
```

Comment:
"Deploying a service with the [Open Autonomy](https://github.com/valory-xyz/open-autonomy) framework requires that you have a service configuration available already, which could be a service created from scratch,"